rcl/test/rcl/common.hpp

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef RCL__COMMON_HPP_
+#define RCL__COMMON_HPP_
+
+#include "rcl/client.h"
+#include "rcl/service.h"
+#include "rcl/rcl.h"
+
+bool
+wait_for_server_to_be_available(
```

Comment:
Please, add a minimal doc block to each function (it's used just for testing, but it'll help for future maintainers)

Reply:
Okay. Add doc bock for each function.